requirements/specifications/intents/user-interest.md

```diff
@@ -0,0 +1,290 @@
+`xrn:firebolt:capability:discovery:interest` capability in order to 
+listen to the `Content.onUserInterestedIn` notification.
+
+If the result is `null` or is not a valid entity, i.e. does not match
+the [EntityInfo](../entities/) schema, then no `Content.onUserInterestedIn`
+notification will be dispatched.
+
+The `Discovery.userInterest` method **MUST NOT** be used in place of more
+specific Discovery methods, e.g. `Discovery.watchNext` or
+`Discovery.watched`. These methods facilitate specific UX flows that may
+have separate legal opt-outs for each user.
+
+The `Discovery.userInterest` method **MUST NOT** be called unless the user
+is activating a UI element in the app, or in a second screen experience
+that is communicating with the app, that implies interest of some kind.
+
+**TODO**: This seems like a contractual obligation, as testing this is
+non-trivial / impossible. Are we keeping track of contractual
+requirements anywhere?
+
+## 4. User Interest from a platform UX
+
+Firebolt platforms may provide a platform UX, e.g. voice or and RCU, to
+express user interest in content from an active App. To facilitate this
+Apps will need to be told about the user\'s expressed interest in their
+content.
+
+First, the Aggregated Experience (or some app with the correct
+capability) detects that the user is interested in something. In this
+picture the interest is triggered by an RCU button, but how this occurs
+is outside the scope of this document. When this happens, the Aggregated
+Experience app calls `Content.requestUserInterest()`, which will trigger the
+foreground app's UserInterest provider and call it's `userInterest` method
+by invoking the RPC method `Discovery.onRequestUserInterest`.
+
+![](../../../requirements/images/specifications/intents/user-interest/media/image3.png)
+
+Next, the foreground app receives and responds to the request with an 
+EntityInfo, which is returned as the result to the pending
+`Content.requestUserInterest` method:
+
+![](../../../requirements/images/specifications/intents/user-interest/media/image4.png)
+
+A user's intention to express interest in something is handled by the
+`InterestIntent`. This intent happens *before* the platform knows which
+content is displayed and/or selected by the foreground app. The
+`InterestIntent` may originate from a cloud system, e.g. Voice, or from the RCU, or elsewhere. Origination of this intent is out
+of scope for this document. This intent includes the type of interest,
+e.g. interest or disinterest.
+
+To fire an `InterestIntent` to the current App, another app, typically
+the Aggregated Experience, will call the `Content.requestUserInterest()` method
+of the Discovery SDK.
+
+When `Content.requestUserInterest` is called, if the App in the foreground 
+Lifecycle state has registered a provider for the `xrn:firebolt:capability:discovery:interest` capability, then it **MUST** have that provider invoked with the values from the
+`Content.requestUserInterest` call.
```

Comment:
This section (144-157) is sort of repeating what is already in the previous section starting at line 128

Reply:
yeah, this is describing how an upstream system would send a `UserInterest` intent, but it's not very clear, and it is repetitive. will rework.